boefjes/boefjes/plugins/kat_masscan/main.py

```diff
@@ -0,0 +1,110 @@
+import io
+import logging
+import os
+import tarfile
+from typing import ByteString, Generator, List, Tuple, Union
+
+import docker
+
+from boefjes.job_models import BoefjeMeta
+
+IMAGE = "ghcr.io/minvws/nl-kat-masscan-build-image:pr-1340"
+
+
+###############################################################################
+# Identical to Webpage Capture boefje.
+###############################################################################
```

Comment:
Perhaps a separate issue, but it would be nice if some kind of boefje utils would be supported so that code like this to fetch a file from a Docker container can be easily re-used in different boefjes

Reply:
Separate issue but indeed a very good idea. I'm actually looking at using this functionality in other tools that I want to integrate too.

Reply:
https://github.com/minvws/nl-kat-coordination/issues/1343